README.md

```diff
@@ -5,7 +5,32 @@
 [license-img]: http://img.shields.io/badge/license-MIT-brightgreen.svg
 [license]: http://opensource.org/licenses/MIT
 
-Elixir School's Extracurricular is a website and twitter bot intended to increase visibility to open source opportunities within the community.
+Elixir School's Extracurricular is a website and twitter bot intended to increase the visibility of open source opportunities within the community.
+
+## Running the Project
+
+Once you have cloned the repo and `cd`'d in, its quite simple to get started.
+First, we get our dependencies and compile everything:
+
+```bash
+$ mix do deps.get, compile
+```
+
+Next we need to setup our database.  Out-of-the-box Extracurricular uses Postgres with a user `postgres` and password `postgres`.  If you're using anything different you'll need to fill out the proper credentials in `apps/web/config/dev.exs` and `apps/web/config/test.exs`.
+
+To setup the database, migrate it, and populate it with dummy data we can use our handy alias:
+
+```bash
+$ mix ecto.setup
+```
+
+And the fun part, running the server!  If you haven't done so already, this will compile your frontend dependencies:
+
+```bash
+$ iex -S mix phx.server
```

Comment:
Does this work properly without building brunch to compile the assets? Gonna pull local and run through myself on a fresh clone but wondered if you knew off the top of your head.

Reply:
@ybur-yug it's worked for me so far.  I tried blowing things away and it compiled everything just fine.  Let me know once you run it locally 👍